my_config.yaml

```diff
@@ -0,0 +1,7 @@
+sourceFormat: HUDI
```

Comment:
Any reason for the file to be at the top level ? It's causing license check failures.

Reply:
> Any reason for the file to be at the top level ? It's causing license check failures.

the `my_config.yaml` file was unintended for this PR. Removed it now.